SevenSwitch.swift

```diff
@@ -201,12 +221,18 @@ import QuartzCore
     private var isAnimating: Bool = false
     private var userDidSpecifyOnThumbTintColor: Bool = false
     private var switchValue: Bool = false
+    lazy var knobSize: CGSize = {
+        let height = self.frame.height - 2
```

Comment:
And, actually, the value is wrong. Apple uses 3 points as a margin from each side, so it is 6 pt in total, not 2.